### PR TITLE
SQRXR 107 EfficientNet-Lite0 variant + ROC updates

### PR DESCRIPTION
**Pull request by wingman-jr-addon**
Finally, pull in a new model! This one is based off EfficientNet-Lite0. See https://machinethink.net/blog/mobile-architectures/ and https://blog.tensorflow.org/2020/03/higher-accuracy-on-vision-models-with-efficientnet-lite.html. I found a Keras port at https://github.com/qubvel/efficientnet/issues/104 specifically at https://drive.google.com/drive/folders/1w132jTelMohcb15a27IozRvnCvIQbOgX

### CODE REVIEW

**Comment by YegorZaremba** [comment]
Hi @wingman-jr-addon

Great work :rocket: 

I'll try to test it in my machine on/after holidays to give you some feedback
